casepro/msgs/models.py

```diff
@@ -394,11 +394,13 @@ def create(cls, org, user, messages, action, label=None):
         return action_obj
 
     def as_json(self):
-        return {'id': self.pk,
-                'action': self.action,
-                'created_by': self.created_by.as_json(),
-                'created_on': self.created_on,
-                'label': self.label.as_json() if self.label else None}
+        return {
+            'id': self.pk,
+            'action': self.action,
+            'created_by': self.created_by.as_json(full=False),
```

Comment:
Now User follows same pattern as other objects - `as_json()` returns a full representation, `as_json(full=False)` returns just id/name and is typically used when embedded in another object.